src/ui.form-select/index.vue

```diff
@@ -189,11 +189,15 @@
         </template>
 
         <template #empty="{ emptyStyles }">
-          <span v-show="isEmpty" :class="emptyStyles" v-text="config.i18n.listIsEmpty" />
+          <span
+            v-show="isEmpty"
+            :class="emptyStyles"
+            v-text="config.$messages.listIsEmpty || t('uSelect.listIsEmpty')"
+          />
           <span
             v-show="options.length === 0 && !search && !isEmpty"
             :class="emptyStyles"
-            v-text="config.i18n.noDataToShow"
+            v-text="config.$messages.noDataToShow || t('uSelect.noDataToShow')"
```

Comment:
I think it's better to do like this:

```js
const noDataToShow = te('uSelect.noDataToShow') ? t('uSelect.noDataToShow') : config.$messages.noDataToShow;
```

in this case you can store default en messages inside the component config.

and move this construction to computeds:
docs: https://vue-i18n.intlify.dev/api/injection.html#te-key-locale

You can also create some top level func `getTranslation("noDataToShow")` which will do this stuff under the hood.